### DISCUSSION TITLE
Athom 1 ch relay doesn't allow connection

### DISCUSSION THREAD

**Discussion by kgeddes** [arendst/Tasmota#17165] [Q&A]
picked up a:
[Athom 1Ch Inching/Self-locking Relay Board (R01) Configuration for Tasmota (blakadder.com)](https://templates.blakadder.com/athom_R01.html)
Used tasmotizer-1.2 to set Wifi password and template
Shows up in my DHCP:
tasmota-F976DA-5850
BC-FF-4D-F9-76-DA *
192.168.2.132
can't ping or connect.
ANY SUGGESTIONS?  Athom is useless, they just send me a link to github

**Comment by barbudor** [comment]
I would suggest a full erase + reflash using the web installer https://tasmota.github.io/install/ while being properly powered
And keep the serial connected to get the console and insure Tasmota is properly responding on the serial console
(the web installer includes a serial console which can be activated right after the flashing)

**Reply by kgeddes** [reply]
just did that, with no errors, 
still no ping or WebUI.
13 | 2.4GHz Wireless2 |   | Espressif | 192.168.2.132 | BC:FF:4D:F9:76:DA
Reply from 192.168.2.157: Destination host unreachable.

**Reply by barbudor** [reply]
I may have missed something but why are you pinging '.157' while the DHCP says '.132' ?

**Reply by sfromis** [reply]
You say that the device is 192.168.2.132, but then this looks like the wrong IP?
`Reply from 192.168.2.157: Destination host unreachable.`

**Reply by barbudor** [reply]
Hum, forget my comment
Ping reports the host ip, not the destination here

**Reply by kgeddes** [reply]
Using Windows 10 command line
Pinging 192.168.2.132 with 32 bytes of data:
Reply from 192.168.2.157: Destination host unreachable.
Reply from 192.168.2.157: Destination host unreachable.
Reply from 192.168.2.157: Destination host unreachable.
Reply from 192.168.2.157: Destination host unreachable.

Ping statistics for 192.168.2.132:
    Packets: Sent = 4, Received = 4, Lost = 0 (0% loss),

C:\Users\kim>

**Reply by kgeddes** [reply]
192.168.2.157 is my IP

**Comment by joba-1** [comment]
.157 is the ping sender, noticing it cannot reach .132

Am Mi., 23. Nov. 2022 um 00:14 Uhr schrieb sfromis ***@***.***

**Reply by kgeddes** [reply]
tasmota-F976DA-5850 is the device name

**Reply by kgeddes** [reply]
could it be an issue with the template?

**Reply by sfromis** [reply]
Not likely.

Note that some boards do not work on Wifi while being wired up for serial flashing, but only after rewiring for mains power.

**Reply by kgeddes** [reply]
tried that too, powered off the USB connector same result

**Comment by kgeddes** [comment]
[https://templates.blakadder.com/athom_R01.html](url)

**Reply by kgeddes** [reply]
usb is a power only connector

**Comment by joba-1** [comment]
Have you tried to reach it from a different device? Your phone? Some Windows PC have weird firewall stuff going on...

**Reply by kgeddes** [reply]
yep, phone can't find it either.

**Reply by kgeddes** [reply]
2 different apps Tasmotrol, and Tasmota Remota

**Reply by kgeddes** [reply]
I had it working with an Aliexpress $4 unit but it died after a year, so I got a new one from Athom.
Came with Tasmota already installed.  Reviews on their site said it was just what the nerd needed.

**Comment by joba-1** [comment]
Yes, the part looks interesting - and I would not give up on it yet - can be perfectly fine. I had similar issues with my nous plugs - and now I love them :)

But since you tried all else I,m with barbudor now : erase, reflash. And make sure to provide stable power while flashing. 
You can plug in USB (try different power sources, pc is usually not very strong) or provide 3.3 or 5v directly on the marked pins.
Be sure to connect ground of power source with the board _and_ with the usbserial. 
If I really want to be sure I have lots of clean power, I connect a half charged 1S lipo cell directly to 3.3V input. Beware: fully charged the voltage (4.2V) would be too high, better check!

Consider:
* you could communicate with the device to flash it
* the device runs espressifs wlan code to successfully establish a wlan connection
* the dhcp client can request, receive and acknowledge an ip address and send its hostname to the router.
* and then it cannot answer pings (which is still not tasmota code but lowlevel networking code like dhcp).
I bet it would not be easy to reproduce this behaviour if you tried...
